bench/.npmrc

```diff
@@ -0,0 +1,2 @@
+package-lock=false
+save=false
```

Comment:
What role does `save=false` serve?

Reply:
It prevents npm from modifying the `package.json` in this directory.

Reply:
Ah, yes. I see now why this is important.

> ```console
> $ cd bench
> $ npm install sanctuary-type-classes@2.x.x
> ```